bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java

```diff
@@ -202,12 +205,8 @@ public void transformInpatientClaim() throws FHIRException {
 		ItemsComponent eobSoleItem = eob.getItem().get(0);
 		Assert.assertEquals(Date.valueOf(sourceClaim.getDateAdmission()), eobSoleItem.getServicedPeriod().getStart());
 		Assert.assertEquals(Date.valueOf(sourceClaim.getDateDischarge()), eobSoleItem.getServicedPeriod().getEnd());
-		Assert.assertEquals(
-				eob.getProvider()
-						.getReference(),
-				bundle.getFhirResources().stream()
-						.filter(r -> r instanceof Practitioner).map(
-								r -> (Practitioner) r)
+		Assert.assertEquals(eob.getProvider().getReference(),
+				bundle.getFhirResources().stream().filter(r -> r instanceof Practitioner).map(r -> (Practitioner) r)
 						.filter(p -> p.getIdentifier().stream()
```

Comment:
I think you accidentally formatted the entire file (i.e. these are just unintentional whitespace changes, right?) If so, NBD, but please double-check your commits in the future to avoid it. Just makes diffs really noisy.
You might also adjust your Eclipse "Save Action" preferences such that it automatically formats only the lines you've modified in the future.